docs/Project.toml

```diff
@@ -3,8 +3,4 @@ CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 Glob = "c27321d9-0574-5035-807b-f59d2c89b15c"
 Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
-Oceananigans = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
 Rasters = "a3a2b9e3-a471-40c9-b274-f788e487c689"
-
-[compat]
-Oceananigans = "0.80, 0.81, 0.82"
```

Comment:
why?

Reply:
I'd say this should be the same as
https://github.com/tomchor/Oceanostics.jl/blob/5d122486b6737020697d959028c765d46ecb8e11/Project.toml#L13C10-L13C10
?

Reply:
That's exactly the effect it does, along with the addition of the line 

```
pushfirst!(LOAD_PATH, joinpath(@__DIR__, "..")) # add Oceanostics environment
```

The only difference (I think) is that I don't need to write it twice. It automatically pulls it from `Project.toml`.

Reply:
Gotcha!

Are you sure pushing the main project environment has the desired effect even docs call `Pkg.develop` later?

https://github.com/tomchor/Oceanostics.jl/blob/5d122486b6737020697d959028c765d46ecb8e11/.github/workflows/documentation.yml#L25

Reply:
According to my local test (i.e. pretty much just doing locally the same commands that github actions does), it appears to work as intended. The tests are also passing, so I'm assuming it's okay. Is there anything else I gotta test?